ros/angel_system_nodes/angel_system_nodes/task_monitor.py

```diff
@@ -355,11 +361,39 @@ def task_timer_thread(self):
         self.publish_task_state_message()
 
 
+    def monitor_keypress(self):
+        # Collect events until released
+        with keyboard.Listener(on_press=self.on_press) as listener:
+            listener.join()
+
+
+    def on_press(self, key):
+        """
+        Callback function for keypress events. If the right arrow is pressed,
+        the task monitor advances to the next step.
```

Comment:
It would also be a good idea if you could log an info message when the node starts with this info/instructions so that we are prompted that this is a thing (possibly at the beginning of the `monitor_keypress` function above that is threaded).